build.sbt

```diff
@@ -207,13 +207,18 @@ lazy val dockerSettings = Def.settings(
     val sbtTgz = s"sbt-$getSbtVersion.tgz"
     val sbtUrl = s"https://github.com/sbt/sbt/releases/download/v$getSbtVersion/$sbtTgz"
     val millVersion = Dependencies.millVersion.value
+    val binDir = "/usr/local/bin/"
     Seq(
       Cmd("USER", "root"),
       Cmd("RUN", "apk --no-cache add bash git ca-certificates curl maven openssh"),
       Cmd("RUN", s"wget $sbtUrl && tar -xf $sbtTgz && rm -f $sbtTgz"),
       Cmd(
         "RUN",
-        s"curl -L https://github.com/lihaoyi/mill/releases/download/${millVersion.split("-").head}/$millVersion > /usr/local/bin/mill && chmod +x /usr/local/bin/mill"
+        s"curl -L https://github.com/lihaoyi/mill/releases/download/${millVersion.split("-").head}/$millVersion >${binDir}mill && chmod +x ${binDir}mill"
+      ),
+      Cmd(
+        "RUN",
+        s"curl -L https://git.io/coursier-cli > ${binDir}coursier && chmod +x ${binDir}coursier && coursier install scalafmt --install-dir ${binDir} && rm -rf ${binDir}coursier"
```

Comment:
I tried using native coursier `https://git.io/coursier-cli-linux`, but `/bin/sh: cs: not found` occured.
So, `coursier`, JAR version is chosen. 

I assume the error is due to `alpine` image, which does not contain enough libs for native `cs`.